src/routes/docs/tooling/appwriter/+page.markdoc

```diff
@@ -0,0 +1,50 @@
+---
+layout: article
+title: The Appwriter
+description: Learn about the custom Appwriter mechanical keyboard and its specifications
+---
+
+{% only_dark %}
+![The Appwriter](/images/docs/keyboard/dark/appwriter.png)
+{% /only_dark %}
+
+{% only_light %}
+![The Appwriter](/images/docs/keyboard/appwriter.png)
+{% /only_light %}
+
+The Appwriter is an exclusive mechanical keyboard custom-designed by the Appwrite team. It is optimized to improve developer productivity and is specially tuned to use with the Appwrite Console.
+
+The Appwriter uses icons from the Appwrite Console and Docs on specific keys, making memorizing keyboard shortcuts easier. For example, `G` then `D` is the shortcut for navigating to your project's databases, and the icon on the `D` key matches the icon for Appwrite Databases.
+
+# What's in the box {% #whats-in-the-box %}
+
+The box includes the Appwriter keyboard, a USB-C data/charging cable, a 2-in-1 keycap and switch puller, extra PC layout keycaps, and an information card with a QR code. The keyboard comes with Mac layout keys pre-installed for macOS which can also be swapped with the included PC keycaps.
+
+# Specifications {% #specifications %}
+
+|Specification|Details|
+|----|----|
+|Battery|3000mAh rechargeable battery|
+|Design|75% compact design keyboard|
+|Keycaps|84 custom dye sublimated PBT keycaps|
+|Switches|Gateron G Pro yellow pre-lubed switches|
+|Connections|Tri-mode connectivity (2.4Ghz wireless / bluetooth / wired USB-C)|
+|Case|Noise dampening case-foam|
+|RGB backlight|16 Million colors & multiple lighting effects|
+|Switch support|Hot-swappable switches with 3 & 5 pin|
+|Rollover|N-key rollover|
+
+# Appwriter backlight {% #appwriter-backlight %}
```

Comment:
This page is all about The Appwriter, so the heading doesn't need to include "Appwriter". Let's name this "How to change the backlight"